src/components/Callout/Callout.scss

```diff
@@ -189,11 +88,21 @@ $Callout-beak-slant-width: 28px;
   -webkit-transform: rotate(45deg);
   -ms-transform: rotate(45deg);
   transform: rotate(45deg);
+  &.ms-Callout-smallbeak {
+    width: 16px;
+    height: 16px;
+  }
+  .ms-Callout--blue-bg & {
+    background-color: $ms-color-blue;
+  }
 }
 
-.ms-Callout-smallbeak {
-  @extend .ms-Callout-beak;
 
-  width: 16px;
-  height: 16px;
-}
+//= Modifier: hasCloseButton
+//
+.ms-Callout-button--close {
+    position: absolute;
+    right: 0;
+    top: 0;
+    margin-right: 2px;
```

Comment:
For anything with a right or left position, use the mixins to support right-to-left languages. Here it would be `@include margin-right(2px);`